docker/puppetserver-standalone/Dockerfile

```diff
@@ -11,6 +11,7 @@ ENV PATH=/opt/puppetlabs/server/bin:/opt/puppetlabs/puppet/bin:/opt/puppetlabs/b
 ENV PUPPET_HEALTHCHECK_ENVIRONMENT="production"
 ENV PUPPETSERVER_MAX_ACTIVE_INSTANCES=1
 ENV PUPPETSERVER_MAX_REQUESTS_PER_INSTANCE=0
+ENV ENABLE_CA=true
```

Comment:
Is the point of this PR to enable compile masters, or to simply add a flag for disabling the CA (for whatever purposes)?

I get a sense of both in different parts of the PR, I think.

If the point is to get additional compile masters, then maybe we should name the variables based on that, so we don't need to add additional ENV vars for other things that we need to toggle for compile masters.

For example, maybe this becomes `COMPILE_MASTER=false`, and then in the new ca.sh script we'd check `if [ "$COMPILE_MASTER" = "true" ]; then disable CA`.

This would make it clear in the pupperware compose file too what's going on, and we wouldn't have to add additional ENV vars for other things necessary to get a compile master.

Also thinking about how this looks in the compose file:
```yaml
services:
  puppet:
    image: puppet/puppetserver
  compiler1:
    image: puppet/puppetserver
    environment:
      - COMPILE_MASTER=true
  compiler2:
    image: puppet/puppetserver
    environment:
      - ENABLE_CA=false
      # - SOME_OTHER_ENV_WE_NEED_LATER_FOR_COMPILE_MASTER=true
```

And either way it's easy enough to change later, just wanted to think about the design/naming a bit.

Reply:
Dangit, I missed your commit message 😄 

I see that this PR is not specifically for compile masters, so largely disregard my comment above.

In that case, since we're introducing 2 new variables that go together, I'd suggest naming it `CA_ENABLED` so it matches the related variable `CA_HOSTNAME`, and then any other CA-related settings can follow suit.

Reply:
Yeah, +1 to syncing on `CA_` naming convention.